scripts/esbuild.config.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * NOTE: This file does NOT really generate a bundle version of source-tracking
+ * source-tracking is bundled directly in salesforcedx-vscode
+ * The file is only used to detect any potential risks to esbuild.
+ **/
+const { build } = require('esbuild');
+
+const sharedConfig = {
+  bundle: true,
+  format: 'cjs',
+  platform: 'node',
+  external: [], // The whitelist of dependencies that are not bundle-able
+  keepNames: true,
+  plugins: [],
+  supported: {
+    'dynamic-import': false,
+  },
+  logOverride: {
+    // 'unsupported-dynamic-import': 'error',
+    // dynamic import is used in oclif to refer to a file path, which is legit.
```

Comment:
is it still needed? or explaining why it's disabled?

Reply:
I used to comment this out since there are errors from oclif because of several dynamic imports which works. I just had a try and noticed there are not errors. So I added this check back. Thanks!